downloadDirectory/src/main/resources/run_protege_exporter.pl

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/perl
+
+use GKB::WebUtils;
+
+my $wu = GKB::WebUtils->new_from_cgi();
+my $id = @ARGV[0];
+my $file_name = @ARGV[1];
```

Comment:
you can add `|| die "no id/file_name specified"` to lines 6 and 7 to test the arguments are given